backend/app/api/api_v1/endpoints/expectation.py

```diff
@@ -172,16 +176,38 @@ def delete_expectation(
     )
 
 
-def zip_expectations_and_validations(expectations: list[Expectation], validations):
-
+def zip_expectations_and_validations(expectations: list[Expectation], validations: list[Validation]):
     expectations_as_dict: dict[str, Expectation] = {}
 
     for expectation in expectations:
         expectations_as_dict[expectation.key] = expectation
 
-    for v in validations:
-        source = v["_source"]
-        source["run_date"] = utils.string_to_military_time(source["run_date"])
-        expectations_as_dict[source["expectation_id"]].validations.append(source)
+    for validation in validations:
+        run_time = utils.string_to_military_time(validation.meta.run_id.run_time)
```

Comment:
Never came across "military time" before. Does it mean UTC? 😄

Reply:
Yeah, don't know why I did this. It doesn't look like the `string_to_utc_time` method was used to I updated it to use the `string_to_military_time` implementation.